model/build-processor/src/main/java/org/keycloak/models/map/annotations/IgnoreForEntityImplementationGenerator.java

```diff
@@ -25,7 +25,7 @@
  *
  * @author hmlnarik
  */
-@Retention(RetentionPolicy.SOURCE)
+@Retention(RetentionPolicy.CLASS)
```

Comment:
I was not able to make this work without changing this `RetentionPolicy`. With `RetentionPolicy.SOURCE` [this function](https://github.com/keycloak/keycloak/pull/8948/files#diff-f11d90d4ebaf77241f62c448873d53ce246c65fa54b7fe38cea6132efd7d02d2R107) doesn't work. It is also failing in GHA.